### DISCUSSION TITLE
ACL and TLS features of Redis 6 available?

### DISCUSSION THREAD

**Discussion by anil181** [OT-CONTAINER-KIT/redis-operator#144] [Ideas]
Impressive work! 
I was going through the documentation on https://ot-container-kit.github.io/redis-operator/.

Does this operator support ACL and TLS features of Redis 6 yet?

Thanks & Regards,
Anil

**Comment by iamabhishek-dubey** [comment]
Hi @anil181,

it's in our to-do list.

**Reply by codekoala** [reply]
@iamabhishek-dubey has any of this work been prioritized and/or scheduled?

**Reply by shubham-cmyk** [reply]
@codekoala TLS is already enabled ACL would be fixed soon.

**Comment by shubham-cmyk** [comment]
https://github.com/OT-CONTAINER-KIT/redis-operator/pull/486
ACL
